internal/gateways/vehicle_signal_decoding.go

```diff
@@ -33,12 +33,18 @@ type vehicleSignalDecodingAPIService struct {
 	httpClient shared.HTTPClientWrapper
 }
 
-const VehicleSignalDecodingAPIURL = "https://vehicle-signal-decoding.dimo.zone"
+var VehicleSignalDecodingAPIURL string
 
-func NewVehicleSignalDecodingAPIService() VehicleSignalDecoding {
+func NewVehicleSignalDecodingAPIService(env string) VehicleSignalDecoding {
 	h := map[string]string{}
 	hcw, _ := shared.NewHTTPClientWrapper("", "", 10*time.Second, h, true) // ok to ignore err since only used for tor check
 
+	if env == "dev" {
```

Comment:
@JamesReate  , should we do similar to identity-API?

Reply:
Wonder how identity api for dev and prod are in sync right now

Reply:
yeah this looks good. identity api will need to be the same.

thoughts on making `env` an enum?

Reply:
I added the Environment enum; I put it to vehicleSignalDecodingAPIService, any objections to have it there?